webpack.playground.config.js

```diff
@@ -94,6 +94,7 @@ module.exports = {
       const mockOptions = {
         multiRequest: false,
         proxy: false,
+        quiet: false,
```

Comment:
what does this option do ?

Reply:
display more information to the console where mock server started.
```
okta-signin-widget git:(bacon-topic-add-authenticator-support) ✗ y start:dev-server --no-open
yarn run v1.21.1
$ webpack-dev-server --config webpack.playground.config.js --no-open
Registering POST service at /api/v1/authn/factors
Registering POST service at /api/v1/authn/factors/:factorid/lifecycle/activate
Registering POST service at /api/v1/authn/factors/:factorid/lifecycle/resend
Registering POST service at /api/v1/authn/factors/:factorid/verify
Registering POST service at /api/v1/authn/factors/:factorid/verify/resend
Registering POST service at /api/v1/authn
Registering POST service at /api/v1/authn/skip
Registering POST service at /idp/idx
Registering POST service at /idp/idx/authenticators/okta-verify/launch
Registering POST service at /idp/idx/authenticators/poll
Registering POST service at /idp/idx/authenticators/poll/cancel
Registering POST service at /idp/idx/authenticators/sso_extension/transactions/:transactionId/verify/cancel
Registering POST service at /idp/idx/cancel
Registering POST service at /idp/idx/challenge
Registering POST service at /idp/idx/challenge/answer
Registering POST service at /idp/idx/challenge/poll
Registering POST service at /idp/idx/challenge/resend
Registering POST service at /idp/idx/credential/enroll
Registering POST service at /idp/idx/enroll
Registering POST service at /idp/idx/identify
Registering POST service at /idp/idx/introspect
Registering POST service at /idp/idx/login/token/redirect
Registering POST service at /idp/idx/recover
Registering GET service at /idp/idx/authenticators/sso_extension/transactions/:transactionId/verify
Registering POST service at /idp/idx/authenticators/sso_extension/transactions/:transactionId/verify
Registering GET service at /login/getimage
Registering GET service at /login/sessionCookieRedirect
Project is running at http://localhost:3000/
webpack output is served from /
Content not from webpack is served from /Users/haishengwu/okta-oss/okta-signin-widget/playground, /Users/haishengwu/okta-oss/okta-signin-widget/target, /Users/haishengwu/okta-oss/okta-signin-widget/target/js, /Users/haishengwu/okta-oss/okta-signin-widget/target/css
Hash: 2e0e87fd4708f90dbd12
Version: webpack 3.12.0
Time: 1214ms
                   Asset    Size  Chunks                    Chunk Names
    playground.bundle.js  332 kB       0  [emitted]  [big]  playground.js
playground.bundle.js.map  395 kB       0  [emitted]         playground.js
   [2] multi (webpack)-dev-server/client?http://localhost:3000 ./playground/main.js 40 bytes {0} [built]
   [3] (webpack)-dev-server/client?http://localhost:3000 7.93 kB {0} [built]
   [4] ./node_modules/url/url.js 23.3 kB {0} [built]
   [5] ./node_modules/node-libs-browser/node_modules/punycode/punycode.js 14.7 kB {0} [built]
   [7] ./node_modules/url/util.js 314 bytes {0} [built]
   [8] ./node_modules/querystring-es3/index.js 127 bytes {0} [built]
  [11] ./node_modules/strip-ansi/index.js 161 bytes {0} [built]
  [12] ./node_modules/strip-ansi/node_modules/ansi-regex/index.js 135 bytes {0} [built]
  [13] ./node_modules/loglevel/lib/loglevel.js 7.86 kB {0} [built]
  [14] (webpack)-dev-server/client/socket.js 1.08 kB {0} [built]
  [16] (webpack)-dev-server/client/overlay.js 3.67 kB {0} [built]
  [21] (webpack)/hot nonrecursive ^\.\/log$ 170 bytes {0} [built]
  [23] (webpack)/hot/emitter.js 77 bytes {0} [built]
  [25] ./playground/main.js 1.5 kB {0} [built]
  [26] ./.widgetrc 262 bytes {0} [built]
    + 12 hidden modules
webpack: Compiled successfully.
Resolving response for POST /idp/idx/introspect
Resolving response for POST /idp/idx/challenge/poll
Resolving response for POST /idp/idx/challenge/poll
Resolving response for POST /idp/idx/challenge/poll
```

Reply:
nice !